app/src/main/java/com/chrisarriola/githubrxjava/MainViewModel.java

```diff
@@ -0,0 +1,47 @@
+package com.chrisarriola.githubrxjava;
+
+import android.databinding.ObservableField;
+
+import com.manaschaudhari.android_mvvm.ViewModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Func1;
+import rx.schedulers.Schedulers;
+import rx.subjects.PublishSubject;
+
+import static com.manaschaudhari.android_mvvm.FieldUtils.toObservable;
+
+public class MainViewModel implements ViewModel {
+    public final Observable<List<ViewModel>> repositories;
```

Comment:
Yeah. I had some problems making `Observable<List<VMSubclass>>` work, as Data Binding does not support generics in `@BindingAdapter` methods. Hence, this isn't supported in the first release.

Recently, I found that generics can be used in `@BindingConversion`. As soon as I release the next version, I'll update this.